addons/OPT/dynmarkerverwaltung/fn_clientInit.sqf

```diff
@@ -57,25 +57,14 @@ GVAR(dialogCheck) =
 [
 	{
 		//OPT Karten-Dialog
-		if (!(isNull ((findDisplay 444001) displayCtrl 10007))) then
-		{
-			((findDisplay 444001) displayCtrl 10007) call CFUNC(registerMapControl);
-			[{}, {(isNull ((findDisplay 444001) displayCtrl 10007))}] call CFUNC(waitUntil);	
-		};
-
+		[{((findDisplay 444001) displayCtrl 10007) call CFUNC(registerMapControl)}, {(!(isNull ((findDisplay 444001) displayCtrl 10007)))}] call CFUNC(waitUntil);	
```

Comment:
Wenn man diese Konstrukte auf mehrere Zeilen aufteilen würde, dann würde das ganze übersichtlicher werden. Also so in dem Stil hier:
```
[{
        //  hier Code
    },
    {
        // noch mehr Code
    },
]```